dojo/db_migrations/0047_jira_minimum_severity_default.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.14 on 2020-07-10 08:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    def set_default_to_low(apps, schema_editor):
+        system_settings = apps.get_model('dojo', 'system_settings')
+        try:
+            ss = system_settings.objects.get(id=1)
```

Comment:
I think it should be just `System_Settings.objects.get()` like everywhere else, or does this not work?

Reply:
I don't think it's guaranteed to have id=1 for system settings.

Reply:
In what instance could there be a 2nd row for the global settings?

Reply:
just `.get()` didn't work IIRC, hence the `id=1`. I know it's used elsewhere, but it didn't like it in that case.

Reply:
there shouldn't be more then 1 row indeed, but you won't know for sure the id is going to be 1. someone might have changed settings along the way, ran manual queries. maybe has some migrations of his/her own.

.get() should work. Or .objects.all().first() or something.